### PR TITLE
Set cuts

### PR DESCRIPTION
**Pull request by jbothma**

### CODE REVIEW

File: tests/test_cube.py [hunk 1]
```diff
@@ -55,16 +55,13 @@ def test_facts_basic(self):
 
     def test_facts_basic_filter(self):
         facts = self.cube.facts(cuts='cofog1:"4"')
-        assert facts['total_fact_count'] == 12
+        assert facts['total_fact_count'] == 12, facts
         assert len(facts['data']) == 12, len(facts['data'])
 
-    def test_facts_basic_filter_coerce(self):
```
Comment (jbothma) [inline tests/test_cube.py]: Removing because this is removing casting.

File: babbage/query/cuts.py [hunk 2]
```diff
@@ -27,5 +27,5 @@ def apply(self, q, cuts):
             info.append({'ref': ref, 'operator': operator, 'value': value})
             table, column = self.cube.model[ref].bind(self.cube)
             q = self.ensure_table(q, table)
-            q = q.where(column == cast(value, column.type))
```
Comment (jbothma) [inline babbage/query/cuts.py]: removing cast while adding IN. Don't feel like dealing with casting each value and we're removing cast anyway.
~~We the result of this is a server error with SQL error message when the wrong like~~ the result of this is a server error with SQL error message like

> "status": "error", "message": "(psycopg2.ProgrammingError) operator does not exist: text = integer\nLINE 4: WHERE incexp_labeled.item_code IN (100)) AS anon_1\n                                       ^\nHINT:  No operator matches the given name and argument type(s). You might need to add explicit type casts.\n"

Comment (longhotsummer) [inline babbage/query/cuts.py]: This comment looks like you're drunk!

Shouldn't value be the correct type at this point?

Comment (jbothma) [inline babbage/query/cuts.py]: Nope - the value can be as incorrect as in upstream master - this only happens if the user supplies a cut value that is parsed as one type to compare to a column of another type. https://github.com/Code4SA/babbage/pull/7 will change this error message and exception to a query exception and a 400 response in the HTTP API

I got 99 problems and drunkenness ain't one. I'll supply pull requests for the rest :P

**Pull request merged**